.github/workflows/build.yml

```diff
@@ -292,9 +320,8 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
-  final_newline_lint:
+  check_final_newline:
     runs-on: ubuntu-latest
-    name: Check final newline
```

Comment:
This is both out of scope of this pr, and I'd transition toward the other way around. We already have spaces in checks names due to the build matrix, and they are meant to be human readeable.

Reply:
I don't mind either way, but either all checks should have names of this kind or none should. Only one having one does not make much sense.

Reply:
Sure, please fix it properly or don't touch it :p

Reply:
Well if one check is following one naming convention and 20 another I assume we want the later =p

But sure, I'll add names to all of them

Reply:
done